src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -15,6 +15,12 @@
             "homepage": "https://symfony.com/contributors"
         }
     ],
+    "repositories": [
```

Comment:
These changes are only temporary until the PR is accepted or until PR #22190 is merged...